backup/queries_functions.go

```diff
@@ -856,7 +856,7 @@ func GetExtensions(connectionPool *dbconn.DBConn) []Extension {
 			UNION DISTINCT
 			SELECT e.oid, level + 1, e.extname, e.extnamespace
 			FROM cte JOIN e ON cte.oid = e.objid
-		) SELECT oid, quote_ident(extname) name, quote_ident(nspname) schema
+		) SELECT cte.oid, quote_ident(extname) name, quote_ident(nspname) schema
```

Comment:
add information about this change to the description and align it with 80 characters.

Reply:
> align it with 80 characters

in gpbackup there is no such limit (may be another?)

Reply:
or you about description?

Reply:
> or you about description?

yes

Reply:
Done